crates/turborepo-auth/src/token/mod.rs

```diff
@@ -0,0 +1,89 @@
+/*
+ * Much of the login contained in `config_token` and things related to
+ * conversion of tokens should be removed in the future. This is a stopgap
+ * until we are confident or comfortable with making users re-login.
```

Comment:
At some point in the future we'd consider deprecating or removing the config token loading since at said point in the future there shouldn't be any tokens left in `config.json`